docs/detections/investigation-guide-actions.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[interactive-investigation-guides]]
-== Interactive investigation guides
+== Launch Timeline from investigation guides
 
-IMPORTANT: This feature is available for {stack} versions 8.6.0 and newer. Additionally, interactive investigation guides are not compatible between release versions, so query buttons created in 8.6.0 won't render correctly in {version} and vice versa.
+IMPORTANT: Interactive investigation guides are compatible between {stack} versions 8.7.0 and later. Query buttons created in 8.6.x use different syntax and won't render correctly in later versions, and vice versa.
```

Comment:
Perhaps the IMPORTANT callout could go after the paragraph following it so readers have context first.

Reply:
Sounds good to me, thanks!